io_scene_godot/converters/__init__.py

```diff
@@ -21,13 +21,17 @@
 from .armature import export_armature_node, export_bone_attachment
 from .animation import export_animation_data
 
-
+# TODO: What about Empties which refer to group instances? (in 2.8 Collection Instances)
 BLENDER_TYPE_TO_EXPORTER = {
     "MESH": export_mesh_node,
     "ARMATURE": export_armature_node,
     "CAMERA": export_camera_node,
     "LIGHT": export_light_node,
-    "EMPTY": export_empty_node
+    "EMPTY": export_empty_node,
+    "CURVE": export_mesh_node,
```

Comment:
Have you tried that all of these types node could be exported using the mesh exporting flow?

Reply:
I tried a few exports with each object type in a scene with modifiers and shapekeys applied (when possible) and I didn't notice any problems.